engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -2190,11 +2190,13 @@ class IrToTruffle(
               )
               .asInstanceOf[FramePointer]
             val slotIdx = fp.frameSlotIdx()
-            val readArgNoCheck =
+            val readArgNoCheck0 =
               ReadArgumentNode.build(
                 idx,
                 arg.getDefaultValue.orElse(null)
               )
+            val readArgNoCheck =
+              setLocation(readArgNoCheck0, unprocessedArg.name().location())
```

Comment:
The changes in `ReadArgumentNode` and `IrToTruffle` are smaller than I expected! That is a sign this change will likely not have a much of negative effect.